org.eclipse.xtext.xtext.ui.tests/src/org/eclipse/xtext/xtext/ui/editor/hierarchy/XtextCallHierarchyBuilderTest.xtend

```diff
@@ -221,7 +222,7 @@ class XtextCallHierarchyBuilderTest extends AbstractHierarchyBuilderTest {
 					}
 					Model {
 						grammar: org.eclipse.xtext.ui.tests.editor.hierarchy.CallHierarchyBuilderTestLanguage
-						'name=ID' [295, 7]
+						'name=ID' [305, 7]
```

Comment:
I thought it might be easy to get it to the state again where the value was 295 and see where the difference is.